setup.py

```diff
@@ -16,9 +16,11 @@
     license='ISC',
     description='An app for integrating imgix into Django sites',
     long_description=README,
-    url='https://github.com/pancentric/django-imgix',
+    url='https://github.com/imgix/django-imgix',
     author='Pancentric Ltd',
     author_email='devops@pancentric.com',
```

Comment:
More of a question... do we plan on changing these `author` fields or?

Reply:
IMO I don't think we should. I think it's important to give credit to the original authors even if they will not be involved with the future of the library. For our purposes, I think the `maintainer` and `maintainer_email` fields are completely fine. [[1](https://docs.python.org/2/distutils/setupscript.html#additional-meta-data)]

> Either the author or the maintainer must be identified. If maintainer is provided, distutils lists it as the author in PKG-INFO.